eng/pipelines/device-tests.yml

```diff
@@ -62,13 +62,16 @@ parameters:
       name: $(androidTestsVmPool)
       vmImage: $(androidTestsVmImage)
       demands:
-        - macOS.Name -equals Monterey
+        - macOS.Name -equals Ventura
         - macOS.Architecture -equals x64
   - name: iosPool
     type: object
     default:
-      name: $(iosTestsVmPool)
-      vmImage: $(iosTestsVmImage)
+      name: $(androidTestsVmPool)
```

Comment:
is that intended ?

Reply:
Yes I believe @rmarinho was using this to manage running things on a Ventura agent pool internally.  So as long as the tests are running and passing with this setting there should be no concern :)